src/main/resources/jenkins/plugins/office365connector/WebhookJobProperty/help-macros.html

```diff
@@ -0,0 +1,9 @@
+<div align="help">
+  This section defines additional conditions that must be met to send the notification.
```

Comment:
If this is not clear - let me know so I can provide better explanation...

Reply:
@damianszczepanik can you please provide examples for Macro, the notifications are not getting triggered for the ones we tried.

Reply:
There are a few possibilities:
- add parameter to the build and compare this parameter to given value, eg `${param}` `ONE` so once you set `ONE` as the input for build execution, macro matches
- use predefined variable such as `${BUILD_NUMBER}` but this you need to update each time
- use any of the value that match together `abc` and the value `abc`

Reply:
@outconn is there any wat to speed up merge progress? Some PR are waiting weeks to be reviewed and merged :(

Reply:
@damianszczepanik  we are in the process of releasing the merged changes, Test pass is going on, but the Macro was not working so reached out to you. We will release the current set and take up other PRs. Sorry for the delay, we will be releasing the changes once a month from now on, will keep you updated n schedule.

@syamaraj123

Reply:
I wouldn't release now with the current state, I would add #42 before releasing You are going to break a lot of builds 👍

Reply:
@outconn  I'm happy to have one release per month. Main problem which I see now is that PRs are waiting for review/feedback more than week and then it causes some delay with releasing, code rebase and stabilization.